cpp/modules/deck.gl/core/src/core.h

```diff
@@ -24,82 +24,15 @@
 #include "./arrow/arrow-mapper.h"
 #include "./arrow/row.h"
 #include "./lib/attribute/attribute-manager.h"
-#include "./lib/constants.h"  // {COORDINATE_SYSTEM, PROJECTION_MODE}
```

Comment:
Nit: These were added in anticipation of C++ modules and also good habit of showing actual exports. I get that developers don't like to maintain such things, but my take is that the preprocessor based #include system is very unstructured and dated.

This is based on the philosophy that the external API is the primary element of our design and something we want to control very carefully.

Reply:
While I mostly agree with you, I just don't see us keeping these up to date, and currently ~90% of files "export" a single class/struct.

Once modules are available and we start incorporating them, I think the landscape would have changed quite a bit